src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+    public double getBase1() {
+        return base1;
+    }
+
+    public void setBase1(double base1) {
+        this.base1 = base1;
+    }
+
+    public double getBase2() {
+        return base2;
+    }
+
+    public void setBase2(double base2) {
+        this.base2 = base2;
+    }
+
+    public double getHeight() {
+        return height;
+    }
+
+    public void setHeight(double height) {
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * (base1 + base2) * height;
+    }
+
+    @Override
+    public void moreInfo() {
+        System.out.println("Figure: IsoscelesTrapezoid");
+        System.out.println("Base 1: " + base1 + " units");
+        System.out.println("Base 2: " + base2 + " units");
+        System.out.println("Height: " + height + " units");
+    }

```

Comment:
Use name() for getting String representation of enum constants instead of toString(). If 'color' is an enum, use getColor().name() when printing.